esmvaltool/utils/color_tables/show_color_tables.py

```diff
@@ -0,0 +1,83 @@
+"""Utility script for inspecting ncl color tables."""
+import argparse
+import glob
+import logging
+import os
+
+from jinja2 import Template
+
+from esmvaltool.diag_scripts.shared.plot import __file__ as plot_path
+
+logger = logging.getLogger(__name__)
+
+PATH_TO_COLORTABLES = os.path.join(os.path.dirname(plot_path), "rgb")
+
+TEMPDIR = "/tmp"
```

Comment:
Use https://docs.python.org/2/library/tempfile.html#tempfile.gettempdir

Reply:
@bjoernbroetz Can you please have a look at the comment above? I'm not sure if it's a good idea on '/tmp' to be writable on every system. And what will happen if two users on the same system try to run your program?

Reply:
Fixed in bb22195 . No need more for a temporary directory.